src/components/encoding-pane/property-editor.tsx

```diff
@@ -1,11 +1,16 @@
 import * as React from 'react';
 import * as CSSModules from 'react-css-modules';
 import Form from 'react-jsonschema-form';
-import * as vlSchema from 'vega-lite/build/vega-lite-schema.json';
-import {ActionHandler} from '../../actions/redux-action';
-import {SPEC_FIELD_NESTED_PROP_CHANGE, SpecEncodingAction} from '../../actions/shelf/spec';
-import {ShelfFieldDef, ShelfId} from '../../models/shelf/spec/encoding';
+import {debounce} from 'throttle-debounce';
+import {ActionHandler} from '../../actions';
+import {SPEC_FIELD_NESTED_PROP_CHANGE, SPEC_FIELD_PROP_CHANGE, SpecEncodingAction} from '../../actions/shelf';
+import {ShelfFieldDef, ShelfId} from '../../models/shelf/spec';
 import * as styles from './property-editor.scss';
+import {
+  AXIS_ORIENT_SCHEMA, AXIS_ORIENT_UISCHEMA,
+  AXIS_TITLE_SCHEMA, AXIS_TITLE_UISCHEMA,
+  SCALE_TYPE_SCHEMA, SCALE_TYPE_UISCHEMA, STACK_SCHEMA, STACK_UISCHEMA,
+} from './property-editorSchema';
```

Comment:
filename should have lower cases as Mac is natively case insensitive   => `property-editor-schema`.